.github/workflows/docker-push.yml

```diff
@@ -0,0 +1,83 @@
+---
+name: 'Push Docker Image'
+on:
+  # TODO: remove, only for testing; replace with
+  # workflow_dispatch:
```

Comment:
Let's remove the `push` and change it back to `workflow_dispatch`.  
We can merge this PR so the `docker-push.yml` can appear in the action list.  
After that we can start debugging this workflow file.

Reply:
Agreed, done in https://github.com/runtimeverification/kontrol/pull/913/commits/aa2db2252810d2322637286a2211187699909bb4.